app/views/shared/fields/_note.html.erb

```diff
@@ -46,7 +46,7 @@
         elsif note_type != nil && !note_type.blank?
           notes[note_type.titleize] = note
         else
-          notes['General Note'] = note
+          notes['General Note'.to_s] = note
```

Comment:
I think the magic is in line https://github.com/ucsdlib/damspas/blob/feature/note_missing/app/views/shared/fields/_note.html.erb#L21 for Hash#compare_by_identity: notes.compare_by_identity, which is always use the reference id of the key value (string). So it's always inserting a new key => value pairs to the Hash. Then line https://github.com/ucsdlib/damspas/blob/feature/note_missing/app/views/shared/fields/_note.html.erb#L56 to group them together.
Yes, I think we can change the lines to << insert new notes to the array instead if you think it'll be better. What do you think?